pantos/servicenode/configuration.py

```diff
@@ -347,6 +347,29 @@ def get_plugin_config() -> typing.Dict[str, typing.Any]:
     return config['plugins']
 
 
+def get_blockchains_rpc_nodes() \
```

Comment:
This function is the exact copy as in the validator node. Why does that no go into common? The configs of our Pantos backend components have the same structure, which means, that we could have piece of code rather in common, instead of each service itself.

Reply:
The config schema is defined for each project separately. There is no guarantee that the schemas will always stay in sync. And, in fact, the config schema actually different for the blockchain providers since the VN supports multiple providers while the SN only supports one (plus the fallback providers).

Reply:
I just saw the difference in the code, just a tiny `s` which makes the difference. Here it is `provider`, in the validator node its `providers`.

Please dismiss my comment :)